packages/assets-controllers/src/MultichainBalancesController/Poller.ts

```diff
@@ -0,0 +1,34 @@
+import { PollerError } from './error';
+
+export class Poller {
```

Comment:
Instead of adding this poller, it would be nice if the controller could extend `StaticIntervalPollingController` instead of `BaseController`.  That's how the other polling controllers work, and would be good to have them all using the same polling API.

Reply:
+1 here. Being able to extend the `StaticIntervalPollingController` and allowing polling loops to start/stop based on polling ids can help make things more consistent with the existing implementation.

It also enables us to fine tune polling loops for specific data types (`chainId` for instance) I believe that @adonesky1 was also involved in this initial implementation, which we extended.

Reply:
do you strongly think I should do this change in this PR @bergeron @gambinish? I agree with you and this was part of my plan but I just decided to do it in separate PRs. Let me know what you think